### PR TITLE
remove python 3.5 from base images, increase timeout to make php happy

### PR DESCRIPTION
**Pull request by arroyc**
remove python 3.5 from base images, increase timeout to make php happy

### CODE REVIEW

File: vsts/pipelines/buildimage-bases.yml [hunk 1]
```diff
@@ -15,12 +15,12 @@ trigger:
     - images/build/php
     - images/build/yarn-cache
     - build/
-    - vsts/
```
Comment (gpcastro) [inline vsts/pipelines/buildimage-bases.yml]: why remove vsts from the list? everything that impacts the build should be in the list, right?

Comment (kichalla) [reply vsts/pipelines/buildimage-bases.yml]: I reverted this check for Arjun

File: build/build-buildimage-bases.sh [hunk 2]
```diff
@@ -27,9 +27,6 @@ declare -r PYTHON_IMAGE_PREFIX="$ACR_DEV_NAME/public/oryx/python-build"
 docker build -f $__REPO_DIR/images/build/python/2.7/Dockerfile -t "$PYTHON_IMAGE_PREFIX-2.7:$IMAGE_TAG" $__REPO_DIR
 echo "$PYTHON_IMAGE_PREFIX-2.7:$IMAGE_TAG" >> $BUILD_IMAGE_BASES_ARTIFACTS_FILE
 
-docker build -f $__REPO_DIR/images/build/python/3.5/Dockerfile -t "$PYTHON_IMAGE_PREFIX-3.5:$IMAGE_TAG" $__REPO_DIR
```
Comment (gpcastro) [inline build/build-buildimage-bases.sh]: I could swear I had removed this in my PR

Comment (dorfire) [reply build/build-buildimage-bases.sh]: could have been a bad merge

**Comment by arroyc** [comment]
closing this as we are making base image builds parallel #54

File: vsts/pipelines/buildimage-bases.yml (hunk 1 above)
Comment (arroyc) [reply vsts/pipelines/buildimage-bases.yml]: Thanks @kichalla

**Pull request closed without merging**